packages/react-positioning/src/usePopper.ts

```diff
@@ -401,29 +113,27 @@ export function usePopper(
     if (options.target) {
       overrideTargetRef.current = options.target;
     }
-  }, [options.target, overrideTargetRef]);
+  }, [options.target, overrideTargetRef, containerRef]);
+
+  useIsomorphicLayoutEffect(() => {
+    updatePosition();
+  }, [enabled, resolveFloatingUIOptions, updatePosition]);
+
+  // Add window resize and scroll listeners to update position
   useIsomorphicLayoutEffect(() => {
-    handlePopperUpdate();
+    const win = targetDocument?.defaultView;
+    if (!win) {
+      return;
+    }
+
+    win.addEventListener('resize', updatePosition);
+    win.addEventListener('scroll', updatePosition);
 
     return () => {
-      popperInstanceRef.current?.destroy();
-      popperInstanceRef.current = null;
+      window.removeEventListener('resize', updatePosition);
```

Comment:
shouldn't we use window from context  instead direct global ?

Reply:
you're right, that's a pretty bad oversight, I'll fix it

Reply:
updated in [a6e32ef](https://github.com/microsoft/fluentui/pull/21828/commits/a6e32ef4e2385c16581686e0a520bf4ff0f934b6)